Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Robotic/borg_chassis.yml

```
@@ -0,0 +1,82 @@
+  id: LPPBorgChassisSyndicateSaboteurBattery
+  parent: BorgChassisSyndicateSaboteur
+  suffix: Battery
+  name: syndicate saboteur cyborg
+  description: A streamlined engineering cyborg, equipped with covert modules. Its chameleon projector lets it disguise itself as a Nanotrasen cyborg.
+  components:
+  - type: ContainerFill
+    containers:
+      borg_brain:
+      - PositronicBrain
+      borg_module:
+      - LPPBorgModuleChameleon
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)